.changeset/silly-donkeys-remember.md

```diff
@@ -0,0 +1,5 @@
+---
+"@spectrum-css/clearbutton": minor
+---
+
+Improves the visibility of the clear button disabled variant on static color backgrounds by more closely aligning styles with close button.
```

Comment:
Can you update this as much as possible to detail the selector and property updates so that it's easier for SWC to understand the impact of the changes?